addons/binding/org.openhab.binding.deconz/src/main/java/org/openhab/binding/deconz/internal/handler/DeconzBridgeHandler.java

```diff
@@ -121,6 +121,7 @@ private void parseAPIKeyResponse(AsyncHttpClient.Result r) {
             config.apikey = response[0].success.username;
             Configuration configuration = editConfiguration();
             configuration.put(BindingConstants.CONFIG_APIKEY, config.apikey);
+            logger.info("deCONZ api key: " + config.apikey);
```

Comment:
Passwords/Keys should not be logged. You can log into the OSGI openHAB console and query the configuration instead.

Reply:
I have now removed the log output of the API key and added a hint to the readme file.